Content.Shared/Anomaly/SharedAnomalySystem.cs

```diff
@@ -173,6 +176,10 @@ public void DoAnomalySupercriticalEvent(EntityUid uid, AnomalyComponent? compone
     {
         if (!Resolve(uid, ref component))
             return;
+
+        if (!Timing.IsFirstTimePredicted)
+            return;
+
```

Comment:
Is this about the audio? That should just be PlayPredicted instead as you might get prediction issues if anything on client uses AnomalySupercriticalEvent

Reply:
this wasn't about the audio, rather the event for supercritical would run multiple times on the client and mispredict really bad (for things like the random teleport)